tests/Charge/FirstPaymentChargeBuilderTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Laravel\Cashier\Tests\Charge;
+
+use Laravel\Cashier\Http\RedirectToCheckoutResponse;
+use Laravel\Cashier\Mollie\Contracts\CreateMollieCustomer;
+use Laravel\Cashier\Mollie\Contracts\CreateMolliePayment;
+use Laravel\Cashier\Tests\BaseTestCase;
+use Laravel\Cashier\Tests\Fixtures\User;
+use Mollie\Api\MollieApiClient;
+use Mollie\Api\Resources\Customer;
+use Mollie\Api\Resources\Payment as MolliePayment;
+
+class FirstPaymentChargeBuilderTest extends BaseTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->withPackageMigrations();
+        $customer = new Customer(new MollieApiClient);
+        $customer->id = 'cst_unique_customer_id';
+
+        $this->mock(CreateMollieCustomer::class, function ($mock) use ($customer) {
+            return $mock->shouldReceive('execute')
+                ->andReturn($customer);
+        });
+    }
+
+    /** @test */
+    public function redirectToCheckoutResponse()
+    {
+        $this->withMockedCreateMolliePayment();
+        $owner = factory(User::class)->create();
+        $this->assertEquals(0, $owner->orderItems()->count());
+        $this->assertEquals(0, $owner->orders()->count());
+
+        $item = new \Laravel\Cashier\Charge\ChargeItemBuilder($owner);
+        $item->unitPrice(money(100, 'EUR'));
+        $item->description('Test Item');
+        $chargeItem = $item->create();
```

Comment:
Thinking... Should we rename `create()` to `make()` here? the ChargeItem is not yet being persisted I think?

Reply:
Just checked: it's not being persisted. I feel like it should be `make` here, like with Eloquent. What do you think @ciungulete ?